feat: Implement show-attendance command

avinazz3 opened the pull request:
Add functionality to view attendance records of a person in a specific group.

- Create ShowAttendanceCommand to display attendance data
- Create ShowAttendanceCommandParser to handle command input
- Create AttendanceDisplay model to store attendance data for UI
- Create AttendanceCard UI component to render attendance
- Add FXML layout and CSS styles for attendance display

The command follows the pattern:
show-attendance p/PERSON_NAME g/GROUP_NAME

Closes #102

avinazz3 committed:
feat: Implement show-attendance command

Add functionality to view attendance records of a person in a specific group.

- Create ShowAttendanceCommand to display attendance data
- Create ShowAttendanceCommandParser to handle command input
- Create AttendanceDisplay model to store attendance data for UI
- Create AttendanceCard UI component to render attendance
- Add FXML layout and CSS styles for attendance display

The command follows the pattern:
show-attendance p/PERSON_NAME g/GROUP_NAME

Closes #102

lyhthaddeus committed:
Merge branch 'master' into add-showattendancecommand

avinazz3 committed:
fix: test merge

avinazz3 committed:
Merge remote-tracking branch 'origin/add-showattendancecommand' into add-showattendancecommand

lyhthaddeus committed:
Fix Checkstyle

avinazz3 committed:
fix(README): add new line at EOF

avinazz3 committed:
Merge remote-tracking branch 'origin/add-showattendancecommand' into add-showattendancecommand

zwliew reviewed (changes requested):
It doesn't seem like you're using AttendanceCard and AttendanceDisplay, so you should remove them. That aside, I think this looks good to me!

avinazz3 committed:
refactor: Remove unused UI components for attendance display

Delete AttendanceDisplay.java and AttendanceCard.java as we're using
text-based output in the command result instead of custom UI components.

zwliew reviewed (approved):
lgtm!
